### PR TITLE
[ADD] accounting: doc about OCR

### PR DESCRIPTION
**Pull request by kbr-odoo**

### CODE REVIEW

File: accounting/payables/supplier_bills/ocr.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+=====================================================================
+Digitalize your Vendor Bills with Optical Character Recognition (OCR)
+=====================================================================
+
+Encoding your bills manually can be a time-consuming task. Being able to digitalize them and
+extract the data into your database automatically will allow you to reduce errors, save time and
+have an effortless process.
```
Comment (jubodoo) [inline accounting/payables/supplier_bills/ocr.rst]: Digitize is the right word to use 

I would also remove last sentence it's a repetition 

So will allow you to reduce errors and save time.

File: accounting/payables/supplier_bills/ocr.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Set up the feature
+==================
+
+Go to :menuselection:`Accounting --> Settings --> Bill Digitalization`, and choose whether the bills
+should be processed automatically or manually.
```
Comment (jubodoo) [inline accounting/payables/supplier_bills/ocr.rst]: We should change digitalization in the system if it's that :P that's very frenchie

Not your concern tho...

File: accounting/payables/supplier_bills/ocr.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+.. image:: media/setup_ocr.png
+   :align: center
+   :alt: Activate the feature going to the settings application in Odoo Accounting
+
+Start digitalizing your bills
```
Comment (jubodoo) [inline accounting/payables/supplier_bills/ocr.rst]: Digitizing 

Obviously when it's part of the software leave it but fix titles/sentences :)

**Comment by kbr-odoo** [comment]
@jubodoo done

**Pull request merged**